src/pages/workspaces/migration/WorkspaceItem.test.ts

```diff
@@ -92,7 +132,13 @@ describe('WorkspaceItem', () => {
     expect(screen.queryByText(migrationScheduledTooltipText)).toBeNull();
     await user.click(screen.getByLabelText(migrateButtonText));
 
+    // Confirmation dialog
+    expect(screen.queryByText(/Are you sure you want to migrate workspace/i)).toBeTruthy();
+    const confirmDialog = screen.getByRole('dialog');
```

Comment:
Had to scope to within the dialog because there are multiple "Migrate" buttons on the page.